CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,145 @@
+[![SnapIT](./src/assets/images/SnapIT-logo.png)](https://www.snapit.solutions)
```

Comment:
Remove the SnapIT logo. It should not be on the Code of Conduct page, which should be a plain text document. If this is the only place the logo file is used, it can be deleted.

Reply:
Addressed: d41f3c50ab24b61775fe540f073bd1da3d62198d